app/res/layout-v17/textview_rtl.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:textAlignment="viewStart"
+    android:textDirection="locale" />
```

Comment:
@ShivamPokhriyal curious what does textDirection does here, from screenshots it seems like only thing we need to adjust is alignment ?

Reply:
I used `textDirection` as I had a feeling that it is used to define the direction of the text. Like it would change `him` to `mih` for rtl language, though it's not doing that even when I change it to rtl.

Reply:
yeah agree that we don't need `textDirection` for any static texts. It's mostly used to define cursor direction when entering text.